npm/private/npm_import.bzl

```diff
@@ -758,14 +756,13 @@ _ATTRS_LINKS = dicts.add(_COMMON_ATTRS, {
     "lifecycle_build_target": attr.bool(),
     "lifecycle_hooks_env": attr.string_list(),
     "lifecycle_hooks_execution_requirements": attr.string_list(),
-    "lifecycle_hooks_no_sandbox": attr.bool(default = True),
     "npm_translate_lock_repo": attr.string(),
 })
 
 _ATTRS = dicts.add(_COMMON_ATTRS, {
     "extra_build_content": attr.string(),
     "integrity": attr.string(),
-    "patch_args": attr.string_list(default = ["-p0"]),
+    "patch_args": attr.string_list(),
```

Comment:
why remove this? seems like npm_import users might still rely on it?

Reply:
the public API macro which is documented sets the default; this one is sneaky and undocumented